app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly on my side I didn't have recieve `confirmation code` to my email, please fix it and check again it'll work or not 🙏🏼 

![image](https://github.com/cosywasswa/Foods-App/assets/63915024/52844833-8745-4777-85a0-ed5623fe353d)

Reply:
Dear Next Reviewer The solution to this problem is basically explained in this video so kindly watch this and then you would be able to go through that issue.
Thanks

https://github.com/cosywasswa/Foods-App/assets/119049613/bda714e7-05d4-4c7c-96ec-42825fef05dd